src/Microsoft.Extensions.Logging.AzureAppServices/AzureFileLoggerOptions.cs

```diff
@@ -51,8 +51,8 @@ public class AzureFileLoggerOptions: BatchingLoggerOptions
         }
 
         /// <summary>
-        /// Gets or sets a strictly positive value representing the maximum retained file count or null for no limit.
-        /// Defaults to <c>2</c>.
+        /// Gets or sets a string representing the name of the file used to store the logging information.
```

Comment:
Isn't this actually a filename prefix or something like that? I took a quick look at the code and in at least some circumstances that seems to be the case. But I've never used this provider so I'm not too familiar.

But, your comment is certainly better than what was there before!

Reply:
Well, it's the filename, but it's automatically suffixed by the date, as can be seen in this code snippet (from FileLoggerProvider.cs):

```C#
private string GetFullName((int Year, int Month, int Day) group)
{
    return Path.Combine(_path, $"{_fileName}{group.Year:0000}{group.Month:00}{group.Day:00}.txt");
}
```

Maybe I can add this information in the description.

Reply:
`Gets or sets a string representing the prefix of the file the file name used to store the logging...` ?